content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md

```diff
@@ -98,7 +98,7 @@ kubelet 用来提供服务的 IP 地址（设置为<code>0.0.0.0</code> 表示
 
 <tr>
 <td colspan="2">--allowed-unsafe-sysctls strings</td>
-</tr>  
+</tr>
 <tr>
```

Comment:
Hey @helen-frank,

Could you revert those space changes? Thanks.

Reply:
fixed